### PR TITLE
Netprisma: Add device ID for Netprisma LCUR57 and FCUN69 in modem-manager plugin

### PR DESCRIPTION
**Pull request by MankWangNetprisma**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/modem-manager/modem-manager.quirk [hunk 1]
```diff
@@ -122,3 +122,13 @@ CounterpartGuid = USB\VID_2CB7&PID_D00D
...
+
+# Netprisma LCUR57 firehose prog file
+[PCI\VID_203E&PID_1000]
+Summary = Netprisma LCUR57 firehose prog file
+ModemManagerFirehoseProgFile = prog_firehose_sdx24.mbn
+
+# Netprisma FCUN69 firehose prog file
+[PCI\VID_203E&PID_1001]
+Summary = Netprisma FCUN69 firehose prog file
+ModemManagerFirehoseProgFile = prog_firehose_sdx6x.elf
```
Comment (superm1) [inline plugins/modem-manager/modem-manager.quirk:134]: missing newline EOF

**Comment by superm1** [comment]
Would it be possible to try to add emulation data for the new device?  Ideally an end to end update, but if not that can we get enumeration at least?

File: plugins/modem-manager/modem-manager.quirk (hunk 1 above)
Comment (MankWangNetprisma) [reply plugins/modem-manager/modem-manager.quirk]: Newline added.

**Comment by MankWangNetprisma** [comment]
> Would it be possible to try to add emulation data for the new device? Ideally an end to end update, but if not that can we get enumeration at least?
These new devices have the same enumeration data as Quectel EM160 and RM520. So there is no need to add new data. I have also tested that it can be upgraded normally. Thanks!

**Comment by MankWangNetprisma** [comment]
> Would it be possible to try to add emulation data for the new device? Ideally an end to end update, but if not that can we get enumeration at least?

These new devices have the same enumeration data as Quectel EM160 and RM520. So there is no need to add new data. I have also tested that it can be upgraded normally. Thanks!

**Comment by superm1** [comment]
> > Would it be possible to try to add emulation data for the new device? Ideally an end to end update, but if not that can we get enumeration at least?
> 
> These new devices have the same enumeration data as Quectel EM160 and RM520. So there is no need to add new data. I have also tested that it can be upgraded normally. Thanks!

Ok great 👍

**Review by superm1 — APPROVED** [review]

**Pull request merged**